mahta/CSS/CSS-box-model/styles4.css

```diff
@@ -0,0 +1,38 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(5, 20px);
+  grid-gap: 8px;
+}
+.container div {
+  background-color: red;
+  height: 20px;
+  width: 20px;
+}
+
+.containerY {
+  display: flex;
```

Comment:
Same here! Nice job reading about flexbox.

Reply:
Actually reading both flex and grid layouts was only to fix my problems, I couldn't finish the task without the help. Is this OK?

Reply:
Yes, it is. We will have issues about layout in the future, digging deeper into both.